internal/store/migrations.go

```diff
@@ -591,4 +591,29 @@ var migrations = []migration{
 
 		return nil
 	}},
+	{semver.MustParse("0.12.0"), semver.MustParse("0.13.0"), func(e execer) error {
```

Comment:
nit: this is probably the perfect use-case for a schema dot release
```suggestion
	{semver.MustParse("0.12.0"), semver.MustParse("0.12.1"), func(e execer) error {
```